tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "declaration": true,
```

Comment:
Do we really need the declaration files?

Reply:
it doesn't matter. I don't use typescript for compiling.

Reply:
removed